 __tests__/pet.test.js

```diff
@@ -0,0 +1,247 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+  it("returns an object", () => {
+    expect(new Pet("Ted")).toBeInstanceOf(Object);
+  });
+});
+
+describe("constructor", () => {
+  it("sets the name property", () => {
+    const pet = new Pet("Ted");
+
+    expect(pet.name).toEqual("Ted");
+  });
+});
+
+describe("constructor", () => {
+  it("has an initial age of 0", () => {
+    const pet = new Pet("Ted");
+
+    expect(pet.age).toEqual(0);
+  });
+});
+
+describe("constructor", () => {
+  it("has an initial hunger of 0", () => {
+    const pet = new Pet("Ted");
+
+    expect(pet.hunger).toEqual(0);
+  });
+});
+
+describe("constructor", () => {
+  it("has an initial fitness of 10", () => {
+    const pet = new Pet("Ted");
+
+    expect(pet.fitness).toEqual(10);
+  });
+});
+
+describe("isAlive", () => {
+  it("checks if the pet is alive", () => {
+    const pet = new Pet("Garfield");
+    pet.isAlive;
```

Comment:
This line isnt doing anything as you arent calling the function. Even if you were, the returned value its stored anywhere or used in your tests so this should be removed